### DISCUSSION TITLE
410 Client Error - YouTube

### DISCUSSION THREAD

**Discussion by emresaracoglu** [streamlink/streamlink#4494] [Q&A] [answered]
Hello,

There is a non-stop live stream, but I get the 410 error. What is the reason of this?

Log Info:

`[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
error: Unable to open URL: https://youtube.com/get_video_info (410 Client Error: Gone for url: https://www.youtube.com/get_video_info?video_id=SqHIO2zhxbA&el=detailpage)`

**Comment by bastimeyer** [comment, marked as answer]
> What is the reason of this?

You are (once again) using an outdated version (The YouTube plugin doesn't make any `https://youtube.com/get_video_info` requests anymore)

```
$ streamlink -l debug 'https://www.youtube.com/watch?v=SqHIO2zhxbA'
[cli][debug] OS:         Linux-5.17.4-arch1-1-x86_64-with-glibc2.35
[cli][debug] Python:     3.10.4
[cli][debug] Streamlink: 3.2.0+51.g498b1f90
[cli][debug] Requests(2.27.1), Socks(1.7.1), Websocket(1.3.2)
[cli][debug] Arguments:
[cli][debug]  url=https://www.youtube.com/watch?v=SqHIO2zhxbA
[cli][debug]  --loglevel=debug
[cli][debug]  --player=mpv
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/watch?v=SqHIO2zhxbA
[plugins.youtube][debug] Using video ID: SqHIO2zhxbA
[plugins.youtube][debug] This video is live.
[utils.l10n][debug] Language code: en_US
Available streams: 144p (worst), 240p, 360p, 480p, 720p, 1080p (best)
```

Stop spamming low effort questions.

**Reply by emresaracoglu** [reply]
Sorry. I must have mixed it up. The versions on Windows and Debian are different. I forgot this. You are right.
